app/web/utils/nominatim.ts

```diff
@@ -42,3 +42,21 @@ export const simplifyPlaceDisplayName = (place: NominatimPlace) => {
   }
   return addressParts.join(", ");
 };
+
+export const filterDuplicatePlaces = (places: NominatimPlace[] = []) => {
+  const placeImportance = places.reduce((places, place) => {
```

Comment:
This naming is a tiny bit confusing:
- `places` is shadowed
- `placeImportance` sounds like a `Record<string, number>`. Maybe it should be `deduplicatedPlaces`